### ISSUE TITLE
Having a string with a vestaboard char code in brackets breaks .convert()

### ISSUE THREAD

**Issue by coreyzev** [ShaneSutro/Vestaboard#27]
**Describe the bug**

**Broken**
.convert() is broken:
```python
Formatter().convert('{65}N{65}  5 min')
```
Output: 
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File ".venv\lib\site-packages\vestaboard\formatter.py", line 53, in convert
    converted.append(characters[letter])
KeyError: '{'
```

**Working**

.convertLine() works:
```python
Formatter().convertLine('{65}N{65}  5 min', justify="left")
```
Output: `'[65, 14, 65, 0, 0, 31, 0, 13, 9, 14, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]'`

**Comment by ShaneSutro** [comment]
In a previous version we had the opposite of this problem where the line could not _end_ with a bracketed-character code, so this is likely something similar. I’ll take a look into this.

**Comment by coreyzev** [comment]
I noticed that the code for .convertLine is very different than .convert(), they might need more in common.

**Comment by ShaneSutro** [comment]
@coreyzev I dug into this a bit more! Upon looking, this is not actually a bug, the `.convert()` method was always intended as a sort of "helper" to convert words and phrases more easily into the corresponding character codes. The thinking is that if you need a color character, you likely already know the number that the color maps to, so it doesn't need to be supported when converting.

Example: if I wanted to prefix a line with 3 blue bits, I would (most likely) already know that blue == 67, so I wouldn't really need to convert that part, I would only need to convert the letters that follow it, so I would do something like
```python
finalText = [67, 67, 67] + Formatter().convert(" 5 mins")
# output = [67, 67, 67, 0, 31, 0, 13, 9, 14, 19]
```
This is one of those cases where at the very least there should be better documentation, but there are a few use cases where this would be useful to actually implement what you're suggesting:
1. When converting multiple things on a single line - you likely won't want to use `convertLine` there, though it may be an option depending on the use case, or
2. When the color code has the potential to change as well as the letters based on a variable or condition. There's still a way to do it using the list join method above, but it would be nicer to just be able to pass in the already-supported bracketed numbers and have it parsed correctly, or
3. As demonstrated in your use case, when colors surround a potentially changing text
In your example, you'd need to do something like the below, which just feels unnecessarily verbose to me:
```python
finalText = text = [65] + Formatter.convert('n') + [65] + Formatter.convert(' 5 mins')
# output = [65, 14, 65, 0, 31, 0, 13, 9, 14, 19]
```

I think we could re-write the `convert()` helper to accept character codes pretty easily since the logic already exists. I'll plan on implementing this in v1.1.0.

**Comment by ShaneSutro** [comment]
Released in v1.1.0!
